context.go

```diff
@@ -2191,6 +2221,18 @@ func (c *Context) handleRenames() []error {
 		group.name = name
 	}
 
+	for _, replace := range c.replacements {
+		// TODO(ccross): this probably has data races
+		for _, m := range replace.from.reverseDeps {
+			for i, d := range m.directDeps {
+				if d.module == replace.from {
+					m.directDeps[i].module = replace.to
+				}
+			}
+		}
+
+		atomic.AddUint32(&c.depsModified, 1)
```

Comment:
This can be c.depsModified++, but this is also ignored, since we clear and check depsModified within runMutator, which is called before this function (this would work if we were called by runMutator)